app/javascript/hyacinth_ui_v1/components/users/UserEdit.js

```diff
@@ -59,34 +48,41 @@ function UserEdit() {
   const [manageUsers, setManageUsers] = useState(false);
   const [readAllDigitalObjects, setReadAllDigitalObjects] = useState(false);
   const [manageAllDigitalObjects, setManageAllDigitalObjects] = useState(false);
+  const [canActivate, setCanActivate] = useState(false);
 
-  const { loading, error } = useQuery(
-    GET_USER,
+  const canActivateUser = (subject, subjectAbility, object) => {
```

Comment:
Instead of passing the AuthenticatedUser abilities can be checked using `abilities.can?(action, subject)`.